walker_test.go

```diff
@@ -290,12 +290,20 @@ func TestConvert(t *testing.T) {
 		},
 	}
 
-	gotFile := wlkr.convert(path, nil) // ensuring there is no problems with nil file stats.
+	gotFile, err := wlkr.convert(path, nil) // ensuring there is no problems with nil file stats.
+	if err != nil {
+		t.Fatal("unexpected error:", err)
```

Comment:
Just `t.Fatal(err)`. The fact that it fails a test is already unexpected.

Reply:
```suggestion
		t.Fatal(err)
```